lib/results_test.go

```diff
@@ -102,6 +103,9 @@ func TestResultEncoding(t *testing.T) {
 					URL: rapid.String().Draw(t, "url").(string),
 				}
 
+				want.Error = strings.ReplaceAll(want.Error, "\n", "")
```

Comment:
Can you explain this change?

Reply:
rapid uses random seeds at each test and during fuzzy tests sometimes it uses "\n" chars in strings but decoder misplaces it by "\r\n". Break lines in URL strings is unlikely to occur and rapid doesn't have a way to fix initial seed value to avoid those random errors (https://github.com/flyingmutant/rapid/issues/14).

Reply:
removed